app/uk/gov/hmrc/tariffclassificationfrontend/controllers/CaseController.scala

```diff
@@ -51,6 +51,13 @@ class CaseController @Inject()(verify: RequestActions,
   private val pagesWithStartTabIndexes: Map[CaseDetailPage, Int] = Map(TRADER -> 1000, APPLICATION_DETAILS -> 2000, SAMPLE_DETAILS -> 3000, ATTACHMENTS -> 4000,
     ACTIVITY -> 5000, KEYWORDS -> 6000, RULING -> 7000, APPEAL -> 8000)
 
+  def get(reference: String): Action[AnyContent] = (verify.authenticated andThen verify.casePermissions(reference)) { implicit request =>
+    request.`case`.application.`type` match {
+      case ApplicationType.BTI => Redirect(routes.CaseController.trader(reference))
+      case ApplicationType.LIABILITY_ORDER => Redirect(routes.CaseController.activityDetails(reference))
+    }
+  }
+
```

Comment:
i've created a new `/case/1234` Index endpoint which redirects to the default tab based on the case type. All places that redirected straight to the trader tab now point to this